app/services/set_decline_by_default.rb

```diff
@@ -0,0 +1,29 @@
+class SetDeclineByDefault
+  def initialize(application_form:)
+    @application_form = application_form
+  end
+
+  def call
+    return false unless application_choices.count == 3
+
+    most_recent_offered_or_rejected_date = [application_choices.maximum(:offered_at),
+                                            application_choices.maximum(:rejected_at)]
+                                             .compact.max
+
+    dbd_days = TimeLimitCalculator.new(rule: :decline_by_default, effective_date: most_recent_offered_or_rejected_date).call
+
+    dbd_date = dbd_days.business_days.after(most_recent_offered_or_rejected_date).end_of_day
+
+    application_form.application_choices.all? do |application_choice|
+      application_choice.update(decline_by_default_at: dbd_date)
+    end
```

Comment:
Do we need to wrap the operations in this method in a DB transaction or is it already called with a transaction?

Reply:
we'll probably use `update_all`